api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/core/ticket/registry/TicketRegistryProperties.java

```diff
@@ -101,6 +102,12 @@
     @NestedConfigurationProperty
     private RedisTicketRegistryProperties redis = new RedisTicketRegistryProperties();
 
+    /**
+     * Cassandra registry settings.
+     */
+    @NestedConfigurationProperty
+    private CassandraTicketRegistryProperties cassandra = new CassandraTicketRegistryProperties();
```

Comment:
Please make sure the new category of properties is documented correctly in CAS docs.